src/get-weather.js

```diff
@@ -0,0 +1,30 @@
+import temp from './temperature';
+import domStuff from './dom-stuff';
+import pexelsPhoto from './pexels-api/get-photo';
+
+const loadJson = async (location) => {
+  const apikey = '1cd628f79e26ab86281a0d66e944dc4c';
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${location}&appid=${apikey}`, { mode: 'cors' });
+  if (response.status === 200) {
```

Comment:
[`response.ok`](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) will check all success codes, not only 200.